todo_app/app.py

```diff
@@ -1,59 +1,66 @@
 from flask import Flask, render_template, request, url_for, redirect
-from todo_app.flask_config import Config
-from todo_app.data.item import Item
+from todo_app.data.configs.flask_config import Config
+from todo_app.data.ViewModel import ViewModel
 import todo_app.data.session_items_trello as session
-import todo_app.logger_settings as logger
+import todo_app.data.configs.logger_settings as logger
 
-app = Flask(__name__)
-app.config.from_object(Config)
 
+def create_app():
+    app = Flask(__name__)
+    app.config.from_object(Config())
+    
+    @app.route('/')
+    def index():
+        items = session.get_Items_and_Status()[0]
```

Comment:
Given this function is only used here, we could change it to only return the items, and then we don't need to index it with `[0]`

Reply:
Fixed. Thanks.